c/appendix_a.asciidoc

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017-2021 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+[appendix]
+[[changes_to_opencl]]
+= Changes to OpenCL
+
+Changes to the OpenCL C specifications between successive versions are
+summarized below.
+
+== Summary of changes from OpenCL 3.0
+
+The first non-provisional version of the OpenCL 3.0 specifications was *v3.0.5*.
+
+Changes from *v3.0.5*:
+
+  * Clarified that `memory_scope_all_devices` is supported only for OpenCL C 3.0 or newer.
+  * Defined ULP overflow leniency.
+  * Removed a confusing phrase about kernel argument pointer types.
+  * Clarified usage of feature test macros pre-OpenCL C 3.0.
+  * Clarified relationship between optional core features and extensions.
+  * Deprecated the `+__OPENCL_C_VERSION__+` predefined macro and clarified possible values of the macro for different versions of OpenCL.
+
+Changes from *v3.0.6*:
+
+  * Clarified the argument to *vec_step* is not evaluated.
+  * Improved description for pipe specifier.
+  * Fixed spelling of `-cl-no-signed-zeros`.
```

Comment:
Technically an API change rather than an OpenCL C change.

Reply:
Whoops!  Yes, you're right.  I've moved this line item to the API spec.